doc/source/coding_style/flake8.rst

```diff
@@ -2,12 +2,10 @@
 
 Style Guide Enforcement
 =======================
-This topic describes the use of `Flake8`_ for `PEP8`_ style
+This topic describes the use of `flake8`_ for `PEP 8`_ style
 enforcement and the minimum standards expected. PyAnsys libraries
 are expected to be consistent with these guidelines.
 
-.. _PEP8: https://www.python.org/dev/peps/pep-0008/
-
 Flake8
 ~~~~~~
 `Flake8`_ is a Python tool for enforcing code styling. It is a wrapper
```

Comment:
@jorgepiloto I am assuming that the capital letter "F" is breaking the link and consequently preventing the documentation to build properly.
Indeed this link is not referenced anymore.
```suggestion
`flake8`_ is a Python tool for enforcing code styling. It is a wrapper
```

Reply:
Thanks for the feedback, @MaxJPRey. I still need to fix various hyperlinks and cross references. Wanted to do this before keep enhancing the current docs layout.